src/pages/api/sepana/utils.ts

```diff
@@ -0,0 +1,250 @@
+import axios from 'axios'
```

Comment:
**suggestion** this file might be better placed in `src/lib/sepana.ts` WDYT?

Reply:
yes, i've just done a bunch of restructuring in another branch to add /lib/sepana

https://github.com/jbx-protocol/juice-interface/pull/2884/files

i'll try putting the same changes in this PR